### PR TITLE
Handle NAC edge-case

### PR DESCRIPTION
**Pull request by jacobwegner**
[Handle NAC edge-case](https://trello.com/c/JrFmuhQL)

### CODE REVIEW

**Comment by jacobwegner** [comment]
[Handle NAC edge-case](https://trello.com/c/JrFmuhQL/195-handle-nac-edge-case)

File: src/widgets/NewAlexandriaWidget.vue [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  customizations.
+  */
+  import gql from 'graphql-tag';
+
+  import WIDGETS_NS, {
+    NewAlexandriaWidget,
+  } from '@scaife-viewer/scaife-widgets';
+
+  export default {
+    extends: NewAlexandriaWidget,
```
Comment (paltman) [inline src/widgets/NewAlexandriaWidget.vue:21]: This is cool.  Didn't know it was possible to do this.

File: src/widgets/NewAlexandriaWidget.vue [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  This widget contains customizations for Explore Homer that handle
+  special-case queries tothe New Alexandria Commentary API.
+
+  In the future, we may want to take this type of customization
+  into consideration when implementing hooks / extensibility points
+  within `scaife-widgets`.
+
+  We may also find that the Vue Composition API
+  (https://composition-api.vuejs.org/) lends itself to these types of
+  customizations.
```
Comment (paltman) [inline src/widgets/NewAlexandriaWidget.vue:12]: Yes!  composition API is going to change everything for us on Scaife.

File: src/widgets/NewAlexandriaWidget.vue [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    },
+    watch: {
+      originalPassage: {
+        immediate: true,
+        handler() {
+          // special-case handling for the Folios version of
+          // the Iliad
+          if (
+            this.originalPassage.version ===
+            'urn:cts:greekLit:tlg0012.tlg001.msA-folios:'
```
Comment (paltman) [inline src/widgets/NewAlexandriaWidget.vue:50]: Just a minor preference here but not a blocker.  I have found that I like to put calculated conditions like this as computed properties to make it easier to troubleshoot things using Vue DevTools.

```js
    needsHealing() {
       return this.originalPassage.version === 'urn:cts:greekLit:tlg0012.tlg001.msA-folios:';
    },
```

```js
    if (this.needsHealing) {
      this.healFolioURN();
    } else {
```

Comment (jacobwegner) [reply src/widgets/NewAlexandriaWidget.vue:50]: Thanks; have implemented the computed prop.

**Pull request merged**